<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tables</class>
 <widget class="QDialog" name="Tables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose tables from DB</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="showDB_button">
       <property name="text">
        <string>Show tables</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="select_from_table_button">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Show table content</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Custom_Query_Button">
     <property name="text">
      <string>Custom query</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="statusLine">
     <property name="text">
      <string>THIS IS A 'TABLES' WIDGET</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
